go.mod

```diff
@@ -29,6 +28,7 @@ require (
 require (
 	cosmossdk.io/math v1.0.0-beta.4
 	github.com/ory/dockertest/v3 v3.9.1
+	github.com/tendermint/tendermint v0.34.24
```

Comment:
```suggestion
```

Reply:
This was the go mod action, but have updated this time. 
We shouldn't mangle too much what go mod kicks out.